spec/models/registration_spec.rb

```diff
@@ -597,7 +597,9 @@
     it 'positive registration_payment calls registration.consider_auto_close' do
       competition = create(:competition)
       reg = create(:registration, competition: competition)
-      expect(reg).to receive(:consider_auto_close)
+      expect_any_instance_of(Registration).to receive(:consider_auto_close) do |instance|
```

Comment:
This still has `expect_any_instance_of` which I am still vetoing :P
Am I missing something here since you said "Tests have been fixed"?